pkg/database/token.go

```diff
@@ -113,7 +113,12 @@ func (t *Token) Subject() *Subject {
 func (db *Database) ClaimToken(realmID uint, tokenID string, subject *Subject) error {
 	return db.db.Transaction(func(tx *gorm.DB) error {
 		var tok Token
-		if err := db.db.Set("gorm:query_option", "FOR UPDATE").Where("token_id = ? and realm_id = ?", tokenID, realmID).First(&tok).Error; err != nil {
+		if err := tx.
```

Comment:
Note: this wasn't using the transaction before, fixed that.

Reply:
thanks!